File: src/allProject/FamilyGroup/MemberFamily.java
package allProject.FamilyGroup;

import java.util.Objects;

public abstract class MemberFamily {
    private String firstName;
    private String secondName;
    private String nameFather;
    private String nameMother;
    private int age;

    public MemberFamily() {
    }

    public MemberFamily(String firstName, String secondName, String nameFather, String nameMother, int age) {
        this.firstName = firstName;
        this.secondName = secondName;
        this.nameFather = nameFather;
        this.nameMother = nameMother;
        this.age = age;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getSecondName() {
        return secondName;
    }

    public void setSecondName(String secondName) {
        this.secondName = secondName;
    }

    public String getNameFather() {
        return nameFather;
    }

    public void setNameFather(String nameFather) {
        this.nameFather = nameFather;
    }

    public String getNameMother() {
        return nameMother;
    }

    public void setNameMother(String nameMother) {
        this.nameMother = nameMother;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    @Override
    public String toString() {
        return "allProject.FamilyGroup.MemberFamily{" +
                "firstName='" + firstName + '\'' +
                ", secondName='" + secondName + '\'' +
                ", nameFather='" + nameFather + '\'' +
                ", nameMother='" + nameMother + '\'' +
                ", age=" + age +
                '}';
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        MemberFamily that = (MemberFamily) o;
        return age == that.age && Objects.equals(firstName, that.firstName) && Objects.equals(secondName, that.secondName) && Objects.equals(nameFather, that.nameFather) && Objects.equals(nameMother, that.nameMother);
    }

    @Override
    public int hashCode() {
        return Objects.hash(firstName, secondName, nameFather, nameMother, age);
    }
}

File: PythonOOP/FamilyPack/MemberFamily.py
class MemberFamily:

    def __init__(self, first_name, second_name, name_father, name_mother, age):
        self._first_name = first_name
        self._second_name = second_name
        self._name_father = name_father
        self._name_mother = name_mother
        self._age = age

    @property
    def first_name(self):
        return self._first_name

    @first_name.setter
    def first_name(self, value):
        self._first_name = value

    @property
    def second_name(self):
        return self._second_name

    @second_name.setter
    def second_name(self, value):
        self._second_name = value

    @property
    def name_father(self):
        return self._name_father

    @name_father.setter
    def name_father(self, value):
        self._name_father = value

    @property
    def name_mother(self):
        return self._name_mother

    @name_mother.setter
    def name_mother(self, value):
        self._name_mother = value

    @property
    def age(self):
        return self._age

    @age.setter
    def age(self, value):
        self._age = value

    def __str__(self):
        return str(self.first_name) + '\n' + \
            str(self.second_name) + '\n' + \
            str(self.name_father) + '\n' + \
            str(self.name_mother) + '\n' + \
            str(self.age)
